UserAgent.xcodeproj/project.pbxproj

```diff
@@ -6429,7 +6429,7 @@
 			baseConfigurationReference = 8F87D0B7200D772460EF6953 /* Pods-Shared.debug.xcconfig */;
 			buildSettings = {
 				ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES = NO;
-				APPLICATION_EXTENSION_API_ONLY = YES;
+				APPLICATION_EXTENSION_API_ONLY = NO;
```

Comment:
Hope this does not break Today widget

Reply:
I have changed other targets settings to not required api only for extensions. And didn't touch to Today widget.

Reply:
Awesome!